cypress/integration/user.spec.js

```diff
@@ -41,17 +35,4 @@ describe('User connects to their Authority', () => {
     // check that ORCID is connected
     cy.get('[data-testid=orcid-info]').contains('https://sandbox.orcid.org/0000-0001-2345-6789');
   });
-
-  it('The user should be able to connect an authority to their profile', () => {
```

Comment:
Denne testen er sikkert utdatert nå ja, fordi man blir nødt til å koble til forfatter før man får navigere til "Min profil". Men da er det kanskje noe kode i User som kan slettes også? Det er vel kode for å koble til en forfatter fra profilen sin (knapp som åpner modalen)?

Reply:
`ButtonModal` i User.tsx kan slettes

Reply:
sant!